portal_ux/wizards/res_config_settings.py

```diff
@@ -0,0 +1,23 @@
+from odoo import models, fields
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = 'res.config.settings'
+
+    allow_portal_on_companies = fields.Boolean(
+        string="Use Company as Portal User",
+        help="Add company partner to the partners to add to portal access")
```

Comment:
Cambiaría por "Add functionality to allow companies (parents partners) to be portal users, by default odoo does not show parent companies on invite to portal wizard"